### PR TITLE
Form revision

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR overhauls the event form.

It addresses issues #15, #5, and #6

### CODE REVIEW

**Review by evz — COMMENTED** [review]
Remind me again why there are two empty migrations? I vaguely remember talking about that.

**Comment by reginafcompton** [comment]
@evz - it's because I screwed up Alembic migrations, when I first created the app.

File: occrp/forms.py [hunk 1]
```diff
@@ -11,13 +11,15 @@ class StoryForm(FlaskForm):
...
+# Upon data entry: we should trim all strings, as they get entered
 class EventForm(FlaskForm):
-    title = StringField('title', validators=[DataRequired()])
-    start_date = StringField('start_date')
-    end_date = StringField('end_date')
-    description = StringField('description')
-    significance = StringField('significance')
-    person_name = StringField('person_name')
+    description = StringField('description', validators=[DataRequired()])
+    start_date = StringField('start_date', filters=[lambda x: x or None])
```
Comment (evz) [inline occrp/forms.py:17]: These lambdas just ensure that we get a `NULL` in the database (instead of an empty string)?

File: occrp/models.py [hunk 2]
```diff
@@ -65,14 +65,14 @@ class Source(db.Model):
 class Event(db.Model):
     __tablename__ = 'event'
     id = Column(Integer, primary_key=True)
-    title = Column(String, nullable=False, unique=True)
+    description = Column(String, nullable=False, unique=True)
```
Comment (evz) [inline occrp/models.py]: It doesn't look like the unique constraint for the description column is part of the migration. Is that on purpose? Do you care if that's actually checked at the database level?

**Review by evz — CHANGES_REQUESTED** [review]
Just a couple things to take a look at. Other than that looking good.

File: occrp/forms.py (hunk 1 above)
Comment (reginafcompton) [reply occrp/forms.py:17]: Yup, that's what they do. Now, we will not shove empty strings into the database. Addresses this issue: https://github.com/datamade/occrp-timeline-tool/issues/23

**Comment by reginafcompton** [comment]
@evz - those migrations should not be there: they come as a result of me having needlessly run "alembic revision ..." I am removing them.

**Comment by evz** [comment]
You're talking about the empty migrations? Just make sure that you change the "revises" id in this migration: https://github.com/datamade/occrp-timeline-tool/blob/0ffa98e6fa317622dc60c8012b4e079abfa0f30a/alembic/versions/17fd243e4dc4_make_description_not_nullable_and_unique.py so that it doesn't point to one of the empty migrations.

**Comment by reginafcompton** [comment]
@evz - yup, got it...now, I am recreating my DB, because it is screwed up.

File: occrp/models.py (hunk 2 above)
Comment (reginafcompton) [reply occrp/models.py]: Is it not here? https://github.com/datamade/occrp-timeline-tool/pull/26/files#diff-d6dc455171d519ce278427ff03eeff40 (line 24) Or am I misreading this?

**Review by evz — APPROVED** [review]

**Pull request merged**